bin/apply-ssh.bash

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# This script will run an ansible playbook available in kubespray.
+# It will also install some python dependencies for kubespray in a virtual environment
+# It's not to be executed on its own but rather via `ck8s-kubespray apply-ssh <prefix>`.
+
+set -eu -o pipefail
+shopt -s globstar nullglob dotglob
+
+: "${SSH_KEY_LOCATION:?Missing SSH_KEY_LOCATION}"
```

Comment:
```suggestion
: "${SSH_KEY_LOCATION:?Missing SSH_KEY_LOCATION}"
if [ ! -f "${SSH_KEY_LOCATION}" ]; then
```
Or maybe ansible will fail if it's no file?

Reply:
Yeah, ansible fails if it's not a file. It seems cleaner to put a warning at this point though, so I'll add it